dist/array_extensions.js

```diff
@@ -81,8 +81,25 @@
 		if (isNotNumber) {
 			throw new TypeError('Excepted a number');
 		}
-
 		return this.slice(start, this.length);
 	}
-	
+
+	function first(callback=null) {
+		var length = this.length;
+		var index = 0;
+		var isFunction = typeof callback === 'function';
+
+		if(isFunction) {
+			while(index < length) {
```

Comment:
Why you prefer `while` statement over `for(let index = 0; index < length; index += 1) { ... }`?